### ISSUE TITLE
Tab Highlighting Not Working with Dynamic URLs in UNFOLD Settings

### ISSUE THREAD

**Issue by Andy183** [unfoldadmin/django-unfold#834]
When using dynamic URLs in the UNFOLD "TABS" settings configuration, the active tab is not being highlighted correctly. The tab highlighting doesn't work when navigating between different filtered views of the same model.

Current Configuration:
```
"TABS": [
    {
        "models": [
            "website_app.enquiry",
        ],
        "items": [
            {
                "title": _("All Leads"),
                "link": reverse_lazy("admin:website_app_enquiry_changelist"),
            },
            {
                "title": _("No Assigner"),
                "link": lambda request: str(
                    reverse_lazy("admin:website_app_enquiry_changelist")) + "?assigner__exact=No+Assigner",
            },
        ]
    }
]
```

**What version of Unfold are you using?**

0.4

**What version of Django are you using?**

Django 4.2

**What browser are you using?**

Chrome

**Comment by Ehco1996** [comment]
i draft an pr for this https://github.com/unfoldadmin/django-unfold/pull/811

**Comment by Andy183** [comment]
> i draft an pr for this #811

I got it. Thanks man, You're awesome

**Comment by lukasvinclav** [comment]
Fixed in #876
